libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFSmartStore/SFSmartStorePlugin.m

```diff
@@ -387,14 +368,139 @@ - (void)pgGetSoupSpec:(CDVInvokedUrlCommand *)command {
     } command:command];
 }
 
+- (void)pgGetAllGlobalStores:(CDVInvokedUrlCommand *)command {
+    [self runCommand:^(NSDictionary* argsDict) {
+        NSArray *allStoreNames = [SFSmartStore allGlobalStoreNames];
```

Comment:
This should be a `copy` of the array, to avoid threading issues during iteration.